android/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
   // TODO remove this library when default LinearLayoutManager will be fixed.
   compile 'org.solovyev.android.views:linear-layout-manager:0.5@aar'
   compile 'com.timehop.stickyheadersrecyclerview:library:0.4.3@aar'
+  //Glide
+  compile 'com.github.bumptech.glide:glide:3.7.0'
```

Comment:
1. @ygorshenin I confirmed information according the licence compliance with @burivuh . Seems like we just need to add info about this library in copyright list which we already have. FYI: this library includes BSD, part MiT and Apache 2.0 libraries, all these licences are already mentioned in our copyright list.
2. @goblinr Could you please estimate apk size before/after Glide integration and put than info here?